config/yurt-controller/crd/bases/apps.openyurt.io_samples.yaml

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
we should not to maintain yaml files for OpenYurt. please remove yaml files into `charts/templates` dir.

Reply:
These yaml files are generated automatically. How to convert them into chart packages will be discussed later